projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -1041,6 +1041,8 @@ For examples of how to define and use new extension points, please see the code
 ++ speech ++[speechExtPts]
 || Type | Extension Point | Description |
 | ``Action`` | ``speechCanceled`` | Triggered when speech is canceled. |
+| ``Action`` | ``pre_filter_speechSequence`` | Notifies before filtering a speech sequence. |
+| ``Action`` | ``post_filter_speechSequence`` | Notifies when speech sequence has been filtered by NVDA components and/or addons. |
```

Comment:
Here is a suggested alternative phrasing. Separately, I suggest you move this below the line containing `filter_speechSequence`, so the item ordering will follow the processing order.

```suggestion
| ``Action`` | ``post_filter_speechSequence`` | Notifies after a speech sequence has optionally been filtered by NVDA components and/or add-ons. |
```